README.md

```diff
@@ -30,7 +38,7 @@ module.exports = {
         componentLibPath: 'components', // Where your component library lives
         docPagesPath: 'styleguide', // Where your custom styleguide pages live
         basePath: __dirname, // Needed to make above paths relative to your project
-        designSystems: [
+        designSystems: [ // Optional for Parent/Child Design Systems
```

Comment:
This actually isn't true. The build fails if this doesn't exist yet. MVP will be to leave it hardcoded to `/` but ultimately we need to do some finagling with the schema to make this optional.

Reply:
@patrickocoffeyo already uncommented this in the react starter because it breaks the build

Reply:
my bad, I thought this got fixed in your work yesterday